Codecademy/client-modules
# feat(Interstitial): add focus mgmt to Interstitial

### Overview


This PR adds focus management to our Interstitial components. Basically, when an interstitial pops up, the focus will be moved to the heading of that Interstitial. This is considered best practice (as detailed by Gatsby [here](https://www.gatsbyjs.com/blog/2019-07-11-user-testing-accessible-client-routing/)) and mimics the way links works when using screen readers. 

I also made it explicit in the description of the component that only one Interstitial should be rendered on a page at a time. 

You can check out this change's behavior on the PR env:
- [Portfolio Project Interstitial](https://pr-21054.dev-eks.codecademy.com/paths/front-end-engineer-career-path/tracks/fecp-html-css-and-js-portfolio-project/modules/fecp-personal-portfolio-website/kanban_projects/portfolio-website)

### Testing Instructions
1. Visit the above link, or any Portfolio Project you have yet to start. 
2. You should see the intro interstitial.
3. Activate Voiceover (usually cmd + F5).
4. Tab through interstitial using Voiceover + press enter on the Next button.
5. Focus should automatically get pulled up to the title of the Interstitial.
6. Rinse + repeat.
7. ✨


### PR Checklist

- [x] Related to JIRA ticket: [LX-4518](https://codecademy.atlassian.net/browse/LX-4518)
